plugins/cl-who-renderer.lisp

```diff
@@ -0,0 +1,22 @@
+(eval-when (:compile-toplevel :load-toplevel)
+  (asdf:load-system 'cl-who))
+
+(defpackage :coleslaw-cl-who-renderer
+  (:use #:cl #:cl-who)
+  (:import-from #:coleslaw :render-text)
+  (:export #:enable))
+
+(in-package :coleslaw-cl-who-renderer)
+
+(defmethod render-text (text (format (eql :cl-who)))
+  (let* ((*package* (find-package '#:coleslaw-cl-who-renderer))
+         (sexps (with-input-from-string (v text)
+                  (do* ((line (read v)
```

Comment:
I think loop should be used instead of the less popular `DO*`. Something along the lines of 

```lisp
(loop
  :for line := (read v nil 'eof)
  :until (eq line 'eof)
  :do (push line acc)
  :finally (reverse acc))
```